### ISSUE TITLE
[SGE] Phlegma into X Feature

### ISSUE THREAD

**Issue by iRobsie** [Nik-Potokar/XIVSlothCombo#546]
I've just recently started using Sage with the vast majority of the options toggled. However, I noticed in lower leveled instances with "Phlegma into X Feature" enabled, and "Phlegma into Toxikon Feature" disabled (or enabled for that matter) if you do not meet the level requirement for it, it will still try to cast the ability and lock you out of using that one-button combo. I disabled it and it would still try to combo into it regardless. It seems to be more of a "GUI not syncronizing and saving with the actual actions/combos" issue than anything else, I believe.

[Video in reference](https://www.youtube.com/watch?v=GDquQyccY-I)

**Comment by NikkoXIV** [comment]
Can you do a couple things?  First, remove and re-add the button to your hotbar.  Then check and uncheck the boxes for all the options?  And finally, if it's still doing it, try unchecking everything except the Phlegma in X Feature with the options?  

And what were you doing before you entered the dungeon?  Were you doing dungeons on sage, or did you change jobs before you entered?  Were you in PvP or in a PvP zone?

A few of us have tried with the same settings and had no luck replicating it.

**Comment by iRobsie** [comment]
Apologies for the late reply; it was a rare day to get out and go fishing for me. Anyways, I went through and tested what you had suggested and it didn't seem to fix/break it through any of the steps you suggested. 

- I removed/added the button (it almost always tried to default to the Dyskrasia icon after a short period despite me dragging the Phlegma button to the bar).
- I went through and tried it with my current options I've been normally running, and I also tested it with no options enabled but the addon still running.
- I did not attempt leaving only the 'Phlegma into X Feature' enabled but the rest disabled due to the feature seemingly 'fixing itself' and me being unable to find a way to 'break it' again.
- I haven't entered a PVP area in at least two days, and multiple class changes.
- I was previously crafting for a mild amount of time before swapping to Sage to attempt to see if this problem would still occur/was able to recreate the issue, so as such I did change from a crafting job to sage before attempting this.

This issue seemed alot more inconsistant today for me (I haven't really had a chance to play my sage much today, so I can't speak alot on that) but when I was intentionally trying to solve the issue, it randomly started working, and i was unable to 'break it' which is good, but it still shouldn't be breaking from the start. I'll be attaching a video of me trying out said steps and them not working (plus some extras like going into a higher level dungeon, than re-attempting a lower one etc), although it's currently in YT's compression stages at this moment.
[https://www.youtube.com/watch?v=lWLFuTXPlek](https://www.youtube.com/watch?v=lWLFuTXPlek)
